packages/expo-cli/src/credentials/views/IosDistCert.ts

```diff
@@ -202,29 +228,87 @@ export class UseExistingDistributionCert implements IView {
     }
     return null;
   }
+
+  async select(ctx: Context, options: DistCertOptions): Promise<ISelect<DistCert> | DistCert> {
```

Comment:
select and open In most cases should have the same behavior, it would be great to merge those two mechanisms
In all cases open could be implemented using select (if select exists)

I'm assuming that idea of applying changes in db(mentioned on the notion) was droped